CONTRIBUTING.md

```diff
@@ -65,14 +65,59 @@ with `fix:`, `feat:` or similar according to
 
 ## <a name="releasing">Releasing (maintainers only)</a>
 
+### Short version (I've done this before!)
+1. Visit the [GitHub Actions tab](https://github.com/muxinc/media-chrome/actions) and 
+select the "CD" action in the left sidebar.
+2. Click the "Run workflow" dropdown and choose the correct `Version` on the `main` branch.
+   - In the **Use workflow from** select box, make sure **Branch: main** is selected.
+   - In the **Version** select box, choose the appropriate value:
+      - If the commit messages in this release were written using the correct conventional commit style, select `conventional`.  
```

Comment:
praise: good change to not assume you were the author of the commits for this release.